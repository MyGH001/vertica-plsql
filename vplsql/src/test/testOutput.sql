/**
* testing output message with print and dbmd_output 
*/

select PLSQL_CREATE(using parameters content=
$$create or replace procedure p_test_output()
as
begin
  print('Built for Fast.');
  for r IN (select 'Built' as word union all select 'for' union all select 'Freedom.') loop
    dbms_output.put_line(r.word);
  end loop;
end;$$);

select PLSQL_EXEC($$p_test_output()$$);

select PLSQL_DROP(using parameters name='p_test_output');
