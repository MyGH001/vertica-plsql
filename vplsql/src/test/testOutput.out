Ln:1 CREATE PROCEDURE p_test_output
Built for Fast.
Built
for
Freedom.
t
